airbyte-cdk/python/airbyte_cdk/sources/declarative/extractors/record_extractor.py

```diff
@@ -20,7 +19,7 @@ class RecordExtractor:
     def extract_records(
         self,
         response: requests.Response,
-    ) -> List[Record]:
+    ) -> List[Mapping[str, Any]]:
```

Comment:
This is not a breaking change as `Record` was defined as `Mapping[str, Any]` before this PR